npm-shrinkwrap.json

```diff
@@ -8698,7 +8308,7 @@
     },
     "pretty-bytes": {
       "version": "1.0.4",
-      "from": "pretty-bytes@>=1.0.2 <2.0.0",
+      "from": "pretty-bytes@1.0.4",
```

Comment:
Seems to be legit. Maybe some of the new dependencies are directly referencing this particular version?